src/site/stages/build/index.js

```diff
@@ -38,6 +45,50 @@ const rewriteVaDomains = require('./plugins/rewrite-va-domains');
 const updateExternalLinks = require('./plugins/update-external-links');
 const updateRobots = require('./plugins/update-robots');
 
+/**
+ * Immediately copies the Webpack build output to a directory outside of
+ * Metalsmith's build destination, then returns a function for use as a
+ * Metalsmith plugin. This plugin copies the files back to their expected
+ * location.
+ *
+ * This is needed because script/build.sh runs Webpack before the content build,
+ * and Metalsmith's build method removes everything in the destination, which
+ * wipes out the output of the Webpack build.
+ *
+ * This can be removed when we move the content build to a new repository and
+ * this script no longer interacts with the Webpack output at all.
+ */
+function preserveWebpackOutput(metalsmithDestination) {
+  const webpackBuildDirName = 'generated';
+  const tempDir = path.join(__dirname, '../../../../tmp/', webpackBuildDirName);
+  const webpackDir = path.join(metalsmithDestination, webpackBuildDirName);
+
+  const webpackDirExists = fs.existsSync(webpackDir);
+
+  // Immediately move the Webpack output to a new directory
+  if (webpackDirExists) {
+    /* eslint-disable no-console */
+    console.log(`Found Webpack directory at ${webpackDir}`);
+    console.log(
+      'Sanity check -- Does it STILL exist?',
+      fs.existsSync(webpackDir) ? 'yes' : 'no',
+    );
+    console.log('Contents:', fs.readdirSync(webpackDir));
+    fs.moveSync(webpackDir, tempDir, { overwrite: true });
+    /* eslint-enable no-console */
+  }
+
+  return () => {
+    /* eslint-disable no-console */
+    if (webpackDirExists) fs.moveSync(tempDir, webpackDir);
+    else
+      console.log(
+        'No Webpack output found. Skipping the asset preservation step.',
+      );
+    /* eslint-enable no-console */
+  };
+}
+
```

Comment:
When does this execute? Shouldn't it execute at the end of each Webpack build or watch task?

Reply:
Or nvm about the watch task, because I suppose that isn't necessary if to develop for content you need to do a build beforehand

Reply:
Right, so like I explained in the above comment, this is meant to preserve any Webpack output. If there _is_ no Webpack output, then it'll essentially be a no-op.

It's needed only when we run a content build after Webpack does its thing. When we run Webpack by itself with either `yarn build:webpack` or `yarn watch`, we won't need this, like you said.

I'm not doing this at the end of the Webpack build because I'm trying to keep the same output... :thinking: It makes sense in my head, but communicating it is hard...Essentially, I've thought about moving the Webpack output to some directory completely outside `build/`, but we run into other things that need to be changed for that to work seamlessly, and I felt like that was beyond the scope of this particular PR. That'll be coming in a future iteration (if at all).